tests/e2e/src/pages/contacts.page.ts

```diff
@@ -1,8 +1,6 @@
 import { BasePage } from "./base.page";
-import { Helper } from "../helpers/helper";
```

Comment:
This one was removed - because there's no more usage of it on tests/e2e/src/pages/contacts.page.ts
(It is moved to tests/e2e/src/pages/base.page.ts)